front/components/sparkle/AppLayout.tsx

```diff
@@ -519,17 +517,20 @@ function IncidentBanner({ className }: { className?: string }) {
             partial outage.
           </a>
         </span>
-      }
-      label=""
-    >
-      <span>
-        It may cause slowness and errors from assistants using GPT or data
-        retrieval. We are monitoring the situation{" "}
-        <a href="http://status.dust.tt/" target="_blank" className="underline">
-          here
-        </a>
-        .
-      </span>
+        <br />
```

Comment:
```suggestion

```
Would remove it for now for banner display reasons (the banner doesn't display well when on multiple lines). In general, would not use br but if needed div flex col with a gap set to the correct value